app/src/main/java/com/asfoundation/wallet/ui/backup/BackupWalletFragmentPresenter.kt

```diff
@@ -7,25 +7,31 @@ import io.reactivex.disposables.CompositeDisposable
 class BackupWalletFragmentPresenter(private var balanceInteract: BalanceInteract,
```

Comment:
this is not related to this PR but shouldn't we name this class BackupWalletPresenter ? there's an extra Fragment in the name